code/datums/elements/decals/_decal.dm

```diff
@@ -58,22 +62,26 @@
 		"pixel_z" = pixels[2],
 		"smoothing" = smoothing,
 		"cleanable" = cleanable,
-		"desc" = description
+		"desc" = description,
+		"pixel_x" = pixel_x,
+		"pixel_y" = pixel_y
```

Comment:
Pixel shifting decals was added in Wallening, see `"pixel_w" = pixels[1]` and `"pixel_z" = pixels[2]` a few lines up. 

I have a feeling you didn't notice because you can't easily edit `w` and `z` in the map editor (yet?), but in any case it should be atomized out so we can look at this more in depth

Reply:
The wallening is about to be reverted, so no, we gotta do pixel_x/y.

Reply:
We can still use `w/z`, which is why I suggest atomizing it, so we can slap it back in after the revert